modules/perc-distribution-tree/src/main/resources/distribution/rxconfig/Installer/installRepository.xml

```diff
@@ -329,6 +328,19 @@
                        failonerror="true" silenceerrors="false"
         />
 
+        <PSExecSQLStmt printExceptionStackTrace="true" qualifyTableNames="RXS_CT_GENERIC"
+                       qualifyViewNames=""
+                       sqlSqlServer="sp_rename 'RXS_CT_GENERIC.USAGE', 'PUSAGE', 'COLUMN';"
+                       sql="ALTER TABLE RXS_CT_GENERIC rename column USAGE to PUSAGE;"
+                       failonerror="false" silenceerrors="true"
+        />
+<!-- There is an issue altering this table column to not null, thus taking this hack -->
```

Comment:
What happens to the existing data in this table on upgrade?

Reply:
nm I see checking if it is empty.  Won't this still be an issue if the table is not empty?

Reply:
I removed this and used a different approach to handle this case.